x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_items/investigation_items.tsx

```diff
@@ -33,10 +34,14 @@ export function InvestigationItems() {
         />
       </EuiFlexItem>
 
+      {investigation?.id && (
+        <EuiFlexItem grow={false}>
+          <AssistantHypothesis investigationId={investigation.id} />
+        </EuiFlexItem>
+      )}
```

Comment:
🍰 nit: use the context hook `useInvestigation()` directly from AssistantHypothesis:

```suggestion
        <EuiFlexItem grow={false}>
          <AssistantHypothesis />
        </EuiFlexItem>
```

Reply:
I actually had this originally, but it made it so that the investigation was sometimes undefined, and I hated having to handle that all the time. Would you prefer that trade off?